### ISSUE TITLE
Training outputs in the example data - Issue on page /docs/06-train.html

### ISSUE THREAD

**Issue by jccaicedo** [cytomining/DeepProfiler-handbook#16]
It'd be nice to give examples of the outputs we get when training a model on the example data. Here is a couple of items we need:

* A screenshot of the console output after a few epochs.
* Plots of the data stored in the log file (training vs validation loss, accuracy, etc).

Adding that and describing / commenting on those outputs will be useful for people to understand what is going on.

**Comment by Arkkienkeli** [comment]
I have added this.

**Comment by jccaicedo** [comment]
Looks very nice. Great job!
